jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -107,6 +107,39 @@ def html(self):
         )
 
 
+class _InTab(Element):
+    def __init__(
+        self,
+        rawsource="",
+        *children,
+        prefix=JUPYTERLITE_DIR,
+        notebook=None,
+        lite_options={},
+        **attributes,
+    ):
+        app_path = self.lite_app
+        if notebook is not None:
+            lite_options["path"] = notebook
+            app_path = f"{self.lite_app}{self.notebooks_path}"
+
+        options = "&".join(
+            [f"{key}={quote(value)}" for key, value in lite_options.items()]
+        )
+        self.lab_src = f'{prefix}/{app_path}{f"?{options}" if options else ""}'
+
+        super().__init__(
+            "",
+            **attributes,
+        )
```

Comment:
Maybe `rawsource` here ?
```suggestion
        super().__init__(
            rawsource,
            **attributes,
        )
```